paging/paging-testing/build.gradle

```diff
@@ -15,24 +15,41 @@
  */
 
 import androidx.build.LibraryType
+import androidx.build.PlatformIdentifier
 import androidx.build.Publish
 
 plugins {
     id("AndroidXPlugin")
     id("com.android.library")
-    id("org.jetbrains.kotlin.android")
 }
 
-dependencies {
-    api(libs.kotlinStdlib)
-    implementation(project(":paging:paging-common"))
+androidXMultiplatform {
+    jvm()
+    mac()
+    linux()
+    ios()
+    android()
 
-    testImplementation(libs.junit)
-    testImplementation(libs.kotlinCoroutinesTest)
-    testImplementation((libs.kotlinCoroutinesAndroid))
-    testImplementation(project(":internal-testutils-paging"))
-    testImplementation(libs.kotlinTest)
-    testImplementation(libs.truth)
+    defaultPlatform(PlatformIdentifier.JVM)
+
+    sourceSets {
+        jvmMain {
+            dependencies {
+                api(libs.kotlinStdlib)
+                implementation(project(":paging:paging-common"))
+            }
+        }
+        jvmTest {
```

Comment:
Does command `./gradlew paging:paging-testing:connectedCheck` still run successfully? I know we don't have android tests but in paging-common I had to [add android-specific test dependencies](https://android-review.git.corp.google.com/c/platform/frameworks/support/+/2653235/2/paging/paging-common/build.gradle#92) or else `connectCheck` task errors out. Not sure if we need it for paging-testing.

Reply:
I just ran it and it completed successfully